arm/Microsoft.Network/firewallPolicies/deploy.bicep

```diff
@@ -153,6 +153,8 @@ resource firewallPolicy 'Microsoft.Network/firewallPolicies@2021-05-01' = {
   }
 }
 
+// serial deployment is mandatory, otherwise it will fail
```

Comment:
I'd suggest to elaborate more on the reasons why the deployment would fail otherwise. e.g. (please check if it's what you meant in the description)
```suggestion
// Serial deployment is mandatory for multiple rule collection groups to avoid concurrent access to the base policy. 
```

Reply:
CHanged it

Reply:
But I see you changed it completely :D

Reply:
@MrMCake can we resolve the issue?